src/objective.jl

```diff
@@ -160,3 +160,26 @@ function objective_function(model::Model,
                    MOI.ObjectiveFunction{MOIFunType}())::MOIFunType
     return jump_function(model, func)
 end
+
+"""
+    set_objective_coefficient(model::Model, term, coeff)
+
+Updates the objective function to change the coefficient of the given term.
+If the objective does not have the right type, an error will be thrown:
+changing a variable coefficient for a constant objective, changing the
+coefficient of a quadratic term where there is only a linear objective.
+"""
+function set_objective_coefficient end
```

Comment:
You don't need this stub method and docstring.

Reply:
I did that because I wanted this function to also deal with the coefficients of the quadratic terms in the objective. I suspect this will require another method, but I have no idea about how to implement it…

Reply:
We can add the extra method and docstring in the future. For now, just document the method that is implemented.